### PR TITLE
Schedule smoke test against prod every 10 min (Test)

### PR DESCRIPTION
**Pull request by snony14**

### CODE REVIEW

File: .github/workflows/schedule.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Runs e2e every day at 2pm
```
Comment (algreasley) [inline .github/workflows/schedule.yml:1]: This is great .. so simple
(name does not match reality tho 😛 )
Also, I think we should canvass opinion on RT channel .. I don't want this to get in the way of demos, so I think we may want to run it out-of-hours ... 
Could even co-ordinate this with support, who have the responsibility right now of ensuring the RT/RA are running
(ooh, that has given me an idea about some further work for our expanding team 😄)

File: .github/workflows/schedule.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  schedule:
+    - cron: "*/10 * * * *"
+
+defaults:
+  run:
+    working-directory: src/client
+
+jobs:
+  e2e:
+    name: End-to-end test
```
Comment (algreasley) [inline .github/workflows/schedule.yml:12]: mention RT Prod?

File: .github/workflows/schedule.yml (hunk 1 above)
Comment (snony14) [reply .github/workflows/schedule.yml:1]: This is just a sample test  to see if it is going to work for every 10 min😅 . If that's the case, then I will update the schedule.yml and have the cron job run every day at 2pm 😃

File: .github/workflows/schedule.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+name: Runs e2e every day at 2pm against RT Prod
+on:
+  schedule:
+    - cron: "*/10 * * * *"
```
Comment (algreasley) [inline .github/workflows/schedule.yml:4]: as disc, let's run this at 9am to start with, and I think aim it at Dev system until we are happy with the tests and the timing

File: .github/workflows/schedule.yml (hunk 1 above)
Comment (algreasley) [reply .github/workflows/schedule.yml]: ok, def switch to Dev instance then 😅

**Review by algreasley — APPROVED** [review]

**Pull request merged**